mava/components/jax/training/losses.py

```diff
@@ -166,6 +168,252 @@ def policy_loss_fn(
                 )
             return policy_grads, loss_info_policy
 
+        def critic_loss_grad_fn(
```

Comment:
Just maybe chat to @KaleabTessera about this. I think he was thinking one could provide a function to the system instead of having to duplicate the loss code?

Reply:
I'll ask about that. I might have misinterpreted what was being discussed :sweat_smile: .

Reply:
Thanks :)